src/Web/Controllers/AccountController.cs

```diff
@@ -188,7 +188,7 @@ public IActionResult Lockout()
 
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<ActionResult> SignOut()
```

Comment:
any specific reason for changing SignOut to LogOut.

Reply:
Yes, there was a warning about "SignOut" overriding a "SignOut" method in the base controller.  I looked at the latest eShopOnWeb code and noticed it had renamed this to Logout, so I followed suit.